docs/cli/dotnet.md

```diff
@@ -0,0 +1,78 @@
+dotnet command
+==============
+
+## NAME
+
+dotnet -- general driver for running the command-line commands
+
+## SYNOPSIS
+
+dotnet [--version] [--help] [--verbose] < command > [< args >]
+
+## DESCRIPTION
+dotnet is a generic driver for the CLI toolchain. Invoked on its own, it will give out brief usage instructions. 
+
+Each specific feature is implemented as a command. In order to use the feature, it is specified after dotnet, i.e. `dotnet compile`. All of the arguments following the command are command's own arguments.  
+
+
+## OPTIONS
+`-v, --verbose`
+
+Enable verbose output.
+
+`--version`
+
+Print out the version of the CLI tooling
+
+`-h, --help`
+
+Print out a short help and a list of current commands. 
+
+## DOTNET COMMANDS
+
+The following commands exist for dotnet.
+
+* [dotnet-new](dotnet-new.md)
+   * Initializes a sample .NET Core console application. 
+* [dotnet-restore](dotnet-restore.md)
+  * Restores the dependencies for a given application. 
+* [dotnet-build](dotnet-build.md)
+  * Compile the application to either an intermidiate language (IL) or to a native binary. 
+* [dotnet-run](dotnet-run.md)
+   * Runs the application from source.
+* [dotnet-publish](dotnet-publish.md)
+   * Publishes a flat directory that contains the application and its dependencies, including the runtime binaries. 
+* [dotnet-test](dotnet-test.md)
+   * Runs tests using a test runner specified in project.json.
+* [dotnet-pack](dotnet-pack.md)
+   * Create a NuGet package of your code
+
+## EXAMPLES
+
+`dotnew new`
+
+    Initializes a sample .NET Core console application that can be compiled and ran.
+
+`dotnet restore`
+
+    Restores dependencies for a given application. 
+
+`dotnet compile`
+
+    Compiles the application in a given directory. 
+
+## ENVIRONMENT 
+
+`DOTNET_PACKAGES`
+
+    The primary package cache. If not set, defaults to $HOME/.nuget/packages on Unix or %LOCALAPPDATA%\NuGet\Packages (TBD) on Windows.
+
```

Comment:
TBD on what? When? How is the reader of this document to know when it has BD? Will they have to change anything? If they run tools in the in-between-state do they have to themselves keep track of all the binaries on the disk that may be orphaned when this changes?